packages/sequelize-data-provider/src/batchDataProvider.ts

```diff
@@ -17,7 +17,10 @@ export async function loadRelatedEntities(
   keys: readonly GetRelatedEntitiesArgs[],
   getRelatedEntities: (entityName: string, originalEntityId: string, relationFieldName: string, args?: any) => any
 ) {
-  const [keysWithoutArgs, keysWithArgs] = partition(keys, k => isEmpty(k.args));
+  const [keysWithoutArgs, keysWithArgs] = partition(
+    keys,
+    k => isEmpty(k.args) || isEqual(k.args, { skipPagination: true }) || isEqual(k.args, { skipPagination: false })
```

Comment:
extract to meaningful condition name

Reply:
or rather ...i think it's better to remove the skipPaginatioon flag here, instead of the long condition

Reply:
I dont like touching function parameters as they change forever, as we are a util function I should not know what args are used for later ..